src/library/colordelegate.cpp

```diff
@@ -12,23 +12,38 @@ ColorDelegate::ColorDelegate(QTableView* pTableView)
         : TableItemDelegate(pTableView) {
 }
 
-void ColorDelegate::paintItem(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const {
+void ColorDelegate::paintItem(
+        QPainter* painter,
+        const QStyleOptionViewItem& option,
+        const QModelIndex& index) const {
     const auto color = mixxx::RgbColor::fromQVariant(index.data());
 
     if (!color) {
```

Comment:
I'm not quite getting what you refer to.
* if the there's no track color and the row is selected > paint background, either with `QPalette::Highlight` from default palette or `selection-background-color` from stylesheet if available
* if there's a color use it to paint the background, regardless if it's selected or not. That creates consistent appearance for Cover Art and Color in selected rows. I don't see a reason to wash out the color then, the selection is clearly visible with the entire row being painted.

`return` was removed because of
* if the item is also focused > draw a border

Reply:
Done. Just a simple rephrasing.

I had to commit the changes directly. Trying to create a PR resulted in a 404 page.